visallo/visallo
# Unit tests should define graph properties

When running the unit tests, there are currently hundreds of warning messages that look something like:
```
2017-09-25 13:23:56.610/UTC WARN  [org.vertexium.GraphBase] creating default property definition because a previous definition could not be found for property "prop1" of type class java.lang.String
```
This PR doesn't get them all, but it reduces them a significant amount by defining the required properties in a few key places.


Testing Instructions:
Run the Visallo unit tests and ensure that the number of warning messages is reduced.
